packages/frontend/src/translations/en.global.json

```diff
@@ -1439,6 +1439,16 @@
             "resending": "Sending...",
             "resent": "Code sent!",
             "title": "Enter Two-Factor Verification Code"
+        },
+        "action": {
+            "addKey": {
+                "limited": "Adding key ${publicKey} limited to call ${methodNames} on ${receiverId} and spend up to ${allowance} NEAR on gas",
+                "full": "WARNING: Entering the code below will authorize full access to your NEAR account: ${accountId}. If you did not initiate this action, please DO NOT continue.<br /><br />This should only be done if you are adding a new seed phrase to your account. In all other cases, this is very dangerous.<br /><br /> The public key you are adding is: ${publicKey} ",
```

Comment:
nit: Just to avoid confusion here, I think we want to say `if you are adding a new full access key to your account` instead of `if you are adding a new seed phrase to your account` since passphrase is just one way to add a full access key.

Reply:
Good call, i'll update 👍🏽 . I also found this confusing since this message will pop up when disabling 2FA 🤔